django/middleware/csrf.py

```diff
@@ -105,19 +105,38 @@ def _reject(self, request, reason):
         )
         return _get_failure_view()(request, reason=reason)
 
+    def _get_current_token(self, request):
+        try:
+            if settings.CSRF_USE_SESSIONS:
+                return _sanitize_token(request.session[settings.CSRF_COOKIE_NAME])
```

Comment:
Not sure we want to use `settings.CSRF_COOKIE_NAME` for this. What about `CSRF_SESSION_KEY = '_csrf_token'` just like we do with `LANGUAGE_SESSION_KEY`.